### PR TITLE
updated radome and metsensors

### PR DESCRIPTION
**Pull request by ozym**
@elidana @dc735 Here's my latest update from the db. I'm expecting this to be the last time I update these two files.

As these are fairly static I'd suggest we make any changes via a new pull request to this repo rather than trying to force changes into the current db.

I've also updated the tests to help with this.

### CODE REVIEW

File: install/sessions.csv [hunk 1]
```diff
@@ -103,6 +103,7 @@ CORM,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2014-08-21T01:
 CORM,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-09-06T15:25:01Z,2016-11-15T20:47:30Z
 CORM,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2016-11-15T20:47:32Z,9999-01-01T00:00:00Z
 DNVK,GeoNet,GNS,TRIMBLE 5700,GPS,30s,4,linz,trimble_5700 x4,2002-10-01T00:00:00Z,2003-01-28T06:00:00Z
+DNVK,GeoNet,GNS,TRIMBLE 5700,GPS,5s,4,linz,trimble_5700 x4,2003-01-28T06:22:00Z,2003-08-05T00:01:00Z
```
Comment (elidana) [inline install/sessions.csv]: from the data, this should be:
```
DNVK,GeoNet,GNS,TRIMBLE 5700,GPS,30s,4,linz,trimble_5700 x4,2002-10-01T00:00:00Z,2003-01-28T06:22:24Z
DNVK,GeoNet,GNS,TRIMBLE 5700,GPS,5s,4,linz,trimble_5700 x4,2003-01-28T06:22:25Z,2003-08-05T00:01:00Z
```

```
store1.geonet.org.nz:/nv02/gps/raw/2003/028
-r--r--r-- 1 gpsin gpsin  33K Jan 29  2003 DNVK_20030128_060030.dat
-r--r--r-- 1 gpsin gpsin 274K Jan 29  2003 DNVK_20030128_062225.dat
```

File: install/sessions.csv [hunk 2]
```diff
@@ -132,6 +133,7 @@ FRTN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,3,geonet,trimble_netr9 x4,2011-01-18T2
 FRTN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2015-07-07T22:27:39Z,9999-01-01T00:00:00Z
 GDS1,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2016-11-16T00:21:00Z,9999-01-01T00:00:00Z
 GEON,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2007-10-11T20:00:59Z,2007-10-31T00:00:00Z
+GISB,GeoNet,GNS,TRIMBLE 5700,GPS,5s,4,linz,trimble_5700 x5,2002-06-28T00:00:00Z,2002-09-10T22:00:00Z
```
Comment (elidana) [inline install/sessions.csv]: ok (check against the data)

File: install/sessions.csv [hunk 3]
```diff
@@ -152,6 +154,7 @@ GRAC,GeoNet,GNS,ASHTECH Z-XII3,GPS,30s,4,geonet,ashtech_r x6,2000-01-26T00:00:00
 GRAC,GeoNet,GNS,ASHTECH Z-XII3,GPS,30s,4,geonet,ashtech_r x6,2000-02-18T00:00:01Z,2001-01-05T00:00:00Z
 GRAC,GeoNet,GNS,ASHTECH Z-XII3,GPS,30s,4,geonet,ashtech_r x6,2001-01-25T00:00:00Z,2001-01-29T00:00:00Z
 GRAC,GeoNet,GNS,ASHTECH Z-XII3,GPS,30s,4,geonet,ashtech_r x6,2001-01-29T00:00:01Z,2001-01-31T00:00:00Z
+GRAC,GeoNet,GNS,TRIMBLE 4000SSI,GPS,5s,4,geonet,trimble_r x17,2001-01-31T00:00:00Z,2006-08-28T02:05:00Z
```
Comment (elidana) [inline install/sessions.csv]: ok (check against data)

File: install/radomes.csv [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
+Leica Geosystems,LEIS,333,WGTT,2000-06-29T03:00:00Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,405,LYTT,1999-11-15T00:00:00Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,CR13573,HOKI,2000-01-20T00:00:00Z,2003-09-06T00:00:00Z
+Thales,SCIS,0220063995,CNCL,2000-08-03T00:00:00Z,9999-01-01T00:00:00Z
+Thales,SCIS,0220066912,MTJO,2000-08-15T00:00:00Z,9999-01-01T00:00:00Z
 Thales,SCIS,0220067545,WAKA,2000-11-02T00:00:00Z,2001-12-13T00:00:00Z
 Thales,SCIS,0220067548,CNCL,2000-02-05T00:00:00Z,2000-08-03T00:00:00Z
-Thales,SCIS,0220067548,WAKA,2002-05-31T02:44:30Z,2016-03-01T21:13:57Z
-Thales,SCIS,0220073544,NETT,2000-02-07T00:00:00Z,2008-03-18T04:00:00Z
+Thales,SCIS,0220067548,WAKA,2002-05-31T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/radomes.csv]: is there a reason why the undeploy time here is open (and not 2016-03-01 as it was?)

File: install/radomes.csv [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+Thales,SCIS,0220073544,NETT,2000-02-07T00:00:00Z,2008-03-18T04:00:01Z
 Thales,SCIS,0220190193,KARA,2000-02-05T00:00:00Z,2004-12-04T18:30:00Z
-Thales,SCIS,0220190270,QUAR,2000-02-05T00:00:00Z,2015-09-22T23:28:59Z
-Thales,SCIS,0220227659,KARA,2004-12-04T18:31:00Z,2016-03-01T03:15:36Z
-Thales,SCIS,CR519993905,MQZG,2001-09-03T00:00:01Z,2005-02-28T03:00:00Z
+Thales,SCIS,0220190270,QUAR,2000-02-05T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/radomes.csv]: same comment as for WAKA : why closing time is 9999 and not 2016...?

File: install/radomes.csv [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+Thales,SCIS,0220227659,KARA,2004-12-04T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/radomes.csv]: same comment as for WAKA : why closing time is 9999 and not 2016...?

**Comment by ozym** [comment]
It might be easier to merge this as the last auto. And then make a new pull request with what it should be (comments above).

**Comment by elidana** [comment]
yep, fine for me. I've saved the "link" to this pr in the appropriate tickets, so I shall be able to modify those retrospectively

**Pull request merged**